XML/XSD/Common.xsd

```diff
@@ -432,6 +432,17 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="TargetType">
+    <xs:restriction base="xs:token">
+      <xs:enumeration value="None"/>
+      <xs:enumeration value="Self" />
+      <xs:enumeration value="Attacker" />
+      <xs:enumeration value="Group" />
+      <xs:enumeration value="Nearby" />
```

Comment:
Based on previous insanity in other xml, what is the definition of "Nearby"?  Like, I can see this making sense here, but is it numerically defined as, idk: everything in a 3x3 square centered on creature?

Reply:
in this instance, nearby would be limited to the 1x1 square surrounding the creature. Note: currently unused and easily removed if needed.